app/javascript/packs/application-common.js

```diff
@@ -5,11 +5,18 @@
 //
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
+
 import { mount } from '../react/mounter';
 import componentRegistry from '../react/componentRegistry';
+import miqRedux from 'miq-redux';
```

Comment:
I know that webpack does it's job and is smart enough to allow this, but wouldn't it be better to load it trough path rather than from pack?
```typescript
import miqRedux from '../miq-redux';
```

Reply:
Yes, and we should be consistently putting stuff in `ManageIQ.*` in the pack, not in files included from the pack.

That way, we actually use the imports :)